Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [workspace]
-members = ["src/firecracker", "src/jailer", "src/seccompiler", "src/rebase-snap"]
```

Comment:
In my experience it is best to add all packages as workspace members, and I believe this is the recommended and standard approach.

Reply:
I remember now the important reason for this at the moment is that since `bit-fields` is not a part of the `firecracker` or `jailer` dependency chains it means it is excluded from many tests if `bit-fields` is not in the workspace members.